src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/TextareaChoiceListProviderJenkinsTest.java

```diff
@@ -536,4 +536,54 @@ public void testAddEditedValue_CompletedUnstable() throws Exception
             assertFalse("Edited value must not be containd", _testEditedValueWillBeContained(jobname, result, varname, value));
         }
     }
+
+    @Test
+    public void testConfiguration1() throws Exception {

```

Comment:
These two tests reproduce the issue by JEP-200.
I didn't annotate with `Issue` as those tests are not only for JEP-200, but also for generic purposes.
(Actually, those tests should have existed at the beginning)